CONTRIBUTING.md

```diff
@@ -1,39 +1,164 @@
+## Common issues and resolutions
+
+- Using `grape-active_model_serializers`, or any non-Rails server. See
+    [issue](https://github.com/rails-api/active_model_serializers/issues/1258).
+-
 ## How can I help?
 
-Everyone is encouraged to open issues that are affecting you: bugs, ideas, performance problems – everything helps!
+### Filing an issue
+
+Everyone is encouraged to open issues that are affecting you:
+bugs, ideas, documentation, performance problems – everything helps!
+
+#### Before
+
+1. The first place to start is by looking at our [GitHub Issues](https://github.com/rails-api/active_model_serializers/issues).
+
+  - Check if your issue has already reported.
+  - If you find an existing issue report, feel free to add further information to that report.
+
+
+#### Writing
+
+If possible, please include the following information when [reporting an
+issue](https://github.com/rails-api/active_model_serializers/issues/new):
+
+- ActiveModelSerializers version.
+- What are you using ActiveModel Serializers with? Rails? Grape? Other? Which versions?
+- If you are not running the latest version (please check), and you cannot update it,
+  please specify in your report why you can't update to the latest version.
+- Operating system type + version
+- Ruby version with patch level.  And if you're using rvm, rbenv, etc.
+  - Include your ruby -e "puts RUBY_DESCRIPTION".
+- Clearly-written steps to reproduce the issue (i.e. "Show me how to show myself." ), including:
```

Comment:
Maybe state that:
 - best case, provide a failing test
 - if you can't, provide a github repo with a minimal project exhibiting the behavior
 - if you can't, then provide steps to reproduce